README.md

```diff
@@ -63,14 +63,53 @@ The CronJob calls the endpoint `/prisoner-index/queue-housekeeping` which is not
 
 ### Running
 
-`localstack` is used to emulate the AWS SQS and Elastic Search service. When running the integration test this will be started automatically. If you want the tests to use an already running version of `localstack` run the tests with the environment `AWS_PROVIDER=localstack`. This has the benefit of running the test quicker without the overhead of starting the `localstack` container.
+`localstack` is used to emulate the AWS SQS and Elastic Search service.  
+Any commands in `localstack/setup-sns.sh` and `localstack/setup-es.sh` will be run when `localstack` starts, so this contains commands to create the appropriate queues.
+Localstack listens on two main ports - 4566 for sns and sqs and 4571 for elasticsearch.
 
-Any commands in `localstack/setup-sns.sh` and `localstack/setup-es.sh` will be run when `localstack` starts, so this should contain commands to create the appropriate queues.
+Unfortunately localstack needs to be started differently depending on whether you are going to run prisoner offender search in a docker container, or in intellij and in tests.
+If running search in docker `ES_HOSTNAME` needs to be set to `localstack` otherwise it should be set to `localhost`.
+This is because when clients connect it returns a url for subsequent calls and the hostname is then different when in docker versus connecting from a laptop.
 
-Running all services locally:
+The elasticsearch part of localstack takes a long time to start and will only be up and running fully until you see
+```
+[INFO] Running on http://0.0.0.0:4571 (CTRL + C to quit)
+```
+in the localstack logs.
+
+#### Running prisoner offender search in docker
+To start up localstack and other dependencies with prisoner offender search running in docker too:
+```bash
+docker-compose up --scale prisoner-offender-search=0
```

Comment:
won't this bring prisoner-offender-search up and then scale it back down? should this just be 
`docker-compose up`

Reply:
well it won't start prisoner offender search.  We don't want it to be started at this point since localstack isn't up.  The next step does a 
```
docker-compose up
```
to then start it

Reply:
would 
`docker-compose` localstack ouath prisonapi`
be better which would start the three services we want and then
`docker-compose up` which would bring up prisoner-offender-search

Reply:
I don't know - I quite like the `scale` option then it says that we are starting everything apart from that service